### PR TITLE
Remove AWS Device Farm, add info about Firebase Test Lab

### PR DESCRIPTION
**Pull request by helinanever**
- Any references to AWS Device Farm and sylph have been removed
- Added a yaml guide for testing in Firebase Test Lab

### CODE REVIEW

**Comment by helinanever** [comment]
@craastad In Flutter workflow editor docs there's a link to the blog post we have about testing in Firebase Test Lab: https://blog.codemagic.io/codemagic-flutter-integration-tests-firebase-test-lab/. Do you think we should expand the docs on that or is a blog post link enough?

File: content/testing-yaml/firebase-test-lab.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Firebase Test Lab provides cloud-based infrastructure for testing Android and iOS apps on a range of devices and configurations. You can test your apps on Firebase Test Lab as part of the Codemagic build workflow provided you have set up a Firebase project.
+
+To set up testing on Firebase Test Lab in Codemagic, follow these steps.
+
+## Firebase Test Lab prerequisites
+
+You will need to set up the following in the [Firebase console](https://firebase.google.com/).
+
+1. Create a Firebase project.
+2. Create a service account with Editor permissions and create a key. Download the JSON key file so you can authenticate with Firebase Test Lab during the build.
```
Comment (nevercode-kevin) [inline content/testing-yaml/firebase-test-lab.md:16]: Do we have a link to any instructions/existing docs on how to do this?

Comment (helinanever) [reply content/testing-yaml/firebase-test-lab.md:16]: Nothing besides the blog post as far as I know.

Comment (helinanever) [reply content/testing-yaml/firebase-test-lab.md]: Turns out this doc actually describes creating the service account, I've linked to it: https://docs.codemagic.io/knowledge-base/google-play-api/. Will make the file more generic in a separate PR.

**Comment by craastad** [comment]
@helinanever link to the blogpost is fine :+1:

**Review by craastad — APPROVED** [review]
LGTM :+1:
Feel free to squash and merge if you're done making changes. I'm glad this area of the docs got some attention!

**Review by artemii-yanushevskyi — APPROVED** [review]

File: content/testing-yaml/firebase-test-lab.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  --environment-variables coverage=true,coverageFile="/sdcard/coverage.ec" \
+  --directories-to-pull /sdcard
+  --timeout 3m
+```
+
+### iOS
+
+Package your application and prepare it for the upload to Firebase Test Lab as desicribed [here](https://firebase.google.com/docs/test-lab/ios/run-xctest#package-app).
+
+Use generated `MyTests.zip` to start testing:
```
Comment (helinanever) [inline content/testing-yaml/firebase-test-lab.md:62]:
```suggestion
Use the generated `MyTests.zip` to start testing:
```

**Pull request merged**